src/ui/component/viewers/audioVideoViewer.jsx

```diff
@@ -38,7 +42,14 @@ class AudioVideoViewer extends React.PureComponent<Props> {
       sources,
     };
 
-    this.player = videojs(this.videoNode, videoJsOptions, () => {});
+    import(
+        /* webpackChunkName: "videojs" */
+        /* webpackMode: "lazy" */
+        /* webpackPreload: true */
+        'video.js'
+    ).then((videojs) => {
+      me.player = videojs(me.videoNode, videoJsOptions, () => {});
```

Comment:
Can we just use `this.player`?

Reply:
I see it's added in a few places, do we need it?

Reply:
Since there's arrow fn's, yes

Reply:
Arrow fn's don't scope `this`. If it was a regular function you would need it.

Reply:
We have something similar here https://github.com/lbryio/lbry-desktop/blob/master/src/ui/component/fileViewer/internal/player.jsx#L286

Reply:
Seems to work ok with `this`